<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stu_mainwindow</class>
 <widget class="QMainWindow" name="stu_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(215, 233, 237)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="tool_page" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>151</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
    <widget class="QPushButton" name="stu_class_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>我的课程</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stu_task_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>我的作业</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="main_page" native="true">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>88</y>
      <width>591</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>54</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stu_info_list_btn">
       <property name="text">
        <string>人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_user_btn">
       <property name="text">
        <string>切</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stu_exit_btn">
       <property name="text">
        <string>退</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
